src/utils/spo.spec.ts

```diff
@@ -262,6 +262,26 @@ describe('utils/spo', () => {
     });
   });
 
+  it('retrieves tenant app catalog url', async () => {
+    sinon.stub(auth, 'storeConnectionInfo').callsFake(() => Promise.resolve());
+    sinon.stub(request, 'get').callsFake((opts) => {
+      if (opts.url === 'https://graph.microsoft.com/v1.0/sites/root?$select=webUrl') {
+        return Promise.resolve({ webUrl: 'https://contoso.sharepoint.com' });
+      }
+
+      if (opts.url === 'https://contoso.sharepoint.com/_api/SP_TenantSettings_Current') {
+        return Promise.resolve({ CorporateCatalogUrl: 'https://contoso.sharepoint.com/sites/appcatalog' });
+      }
+
+      return Promise.reject('Invalid request');
+    });
+
+    const spoUrl = await spo.getSpoUrl(logger, false);
```

Comment:
Since we're testing retrieving tenant app catalog, we shouldn't be calling this method ourselves as it's not a part of the test case